src/Resources/translations/messages.en.yml

```diff
@@ -13,6 +13,7 @@ sylius_admin_order_creation:
         customer_email: 'Customer email cannot be empty'
         item_discount: 'Item discount'
         new_customer_email: 'New customer email'
+        no_customer_selected: "You haven't selected a customer"
```

Comment:
The message should be placed in single quotes.

Reply:
But it contains a single quote. This would require escaping it and make it less readable.

Reply:
Yup, my bad!